### PR TITLE
TIP-873: Move datagrid bundle to oro

### PR DESCRIPTION
**Pull request by arnolanglade**
**Description (for Contributor and Core Developer)**

The datagrid bundle have been moved to Oro namespace. We don't merged it to OroDatagridBundle because it would have taken lot of time and it is useless.

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | yes
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: CHANGELOG-3.0.md [hunk 1]
```diff
@@ -864,3 +864,4 @@
 - Move `Pim\Bundle\InstallerBundle` to `Akeneo\Platform\Bundle\InstallerBundle`
 - Move `Pim\Bundle\AnalyticsBundle` to `Akeneo\Platform\Bundle\AnalyticsBundle`
 - Move `Pim\Bundle\CatalogVolumeMonitoringBundle` to `Akeneo\Platform\Bundle\CatalogVolumeMonitoringBundle`
+- Move `Pim\Bundle\PimDataGridBundle` to `Oro\Bundle\PimDataGridBundle`
```
Comment (pierallard) [inline CHANGELOG-3.0.md]: Nope

Comment (arnolanglade) [reply CHANGELOG-3.0.md]: Fixed in https://github.com/akeneo/pim-community-dev/pull/8821/commits/4afac92bba8890a044ff82e3f43fccf2a9961366

File: UPGRADE.md [hunk 2]
```diff
@@ -820,3 +820,4 @@ find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Bundle\\UIBundle/Akeneo\\P
 find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Bundle\\ImportExportBundle/Akeneo\\Platform\\Bundle\\ImportExportBundle/g'
 find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Bundle\\AnalyticsBundle/Akeneo\\Platform\\Bundle\\AnalyticsBundle/g'
 find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Bundle\\CatalogVolumeMonitoringBundle/Akeneo\\Platform\\Bundle\\CatalogVolumeMonitoringBundle/g'
+find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Bundle\\PimDataGridBundle/Oro\\Bundle\\PimDataGridBundle/g'
```
Comment (jjanvier) [inline UPGRADE.md]: nope

Comment (arnolanglade) [reply UPGRADE.md]: Fixed in https://github.com/akeneo/pim-community-dev/pull/8821/commits/4afac92bba8890a044ff82e3f43fccf2a9961366

**Pull request merged**